utils/web_scraper.py

```
@@ -1880,8 +1831,740 @@ def format_window_context_content(content: Dict[str, Any]) -> str:
+
+        url = "https://api.bilibili.com/x/polymer/web-dynamic/v1/feed/all"
+        headers = {"User-Agent": get_random_user_agent(), "Referer": "https://t.bilibili.com/"}
+        await asyncio.sleep(random.uniform(0.1, 0.5))
+        
+        async with httpx.AsyncClient() as client:
+            response = await client.get(url, headers=headers, cookies=credential.get_cookies(), timeout=10.0)
+            response.raise_for_status()
+            data = response.json()
+
+        if not isinstance(data, dict) or data.get("code") != 0:
+            logger.error(f"获取B站动态失败，API返回: {data}")
```

Ruff (0.15.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)